inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -177,49 +177,18 @@
                         "title": "Author name",
                         "type": "string"
                     },
-                    "inspire_bai": {
-                        "description": "INSPIRE BAI when available",
-                        "pattern": "(\\w+\\.)+\\d+",
-                        "title": "INSPIRE BibAuthorID",
-                        "type": "string"
-                    },
-                    "inspire_id": {
-                        "description": "Legacy INSPIRE ID when available",
-                        "pattern": "INSPIRE-\\d+",
-                        "title": "INSPIRE ID",
-                        "type": "string"
+                    "ids": {
+                        "items": {
+                            "$ref": "elements/id.json"
+                        },
+                        "type": "array",
+                        "uniqueItems": true
```

Comment:
Note: we might check for the corresponding mapping if it's needed to make this as a nested value. Very likely not, because there is zero chance of conflict of values across different providers (i.e. ORCIDs are very different than JACOW etc.)